### PR TITLE
First pass at making the Audio page responsive

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

The https://audius.co/audio page is not responsive, and gets cut off with no remedy (as we don't have horizontal scrolling) to read the content on smaller screens. Added flexboxes and wrapping to make it responsive.

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

The bottom tier list might have 3 tiers on top, 1 on bottom for certain screen sizes. Is that a dealbreaker? Can put more time into it and force them to break into 2 and 2

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

1. Visit https://marcus.audius.co/audio
2. Resize window slowly, observe responses

⚠️ have not tested mobile!

### CODE REVIEW

File: src/containers/audio-rewards-page/AudioRewardsPage.module.css [hunk 1]
```diff
@@ -15,35 +14,34 @@
...
+  flex-wrap: wrap;
+  justify-content: stretch;
 }
 
 .cryptoContentContainer.mobile {
   flex-wrap: wrap;
 }
 
-.explainerTile {
+.pageContainer .explainerTile {
```
Comment (rickyrombo) [inline src/containers/audio-rewards-page/AudioRewardsPage.module.css:28]: These used to get overwritten by the Tiles.module.css without that specificity.

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/mp-responsive-audio

File: src/containers/audio-rewards-page/components/ExplainerTile.module.css [hunk 2]
```diff
@@ -60,6 +54,7 @@
   font-size: 40px;
   line-height: 49px;
   background: var(--page-header-gradient);
+  background-clip: text;
```
Comment (rickyrombo) [inline src/containers/audio-rewards-page/components/ExplainerTile.module.css:57]: linter complained about not having the non-webkit version of this for compatibility

**Review by raymondjacobson — APPROVED** [review]
Changes LGTM pending julian's ok!

**Pull request merged**